packages/shared-components/src/formio/components/core/driving-list/DrivingList.builder.ts

```diff
@@ -0,0 +1,17 @@
+import DrivingList from './DrivingList';
+
+const drivingListBuilder = () => {
+  const schema = DrivingList.schema();
+  return {
+    title: schema.label,
+    schema: {
+      ...schema,
+      validateOn: 'blur',
+      validate: {
+        required: true,
```

Comment:
Bør endres til false hvis du legger til egen validerings-regel for å hindre at man går videre når vi ikke har noen aktiviteter å vise (se annen kommentar)